src/FSharp.Core/fslib-extra-pervasives.fsi

```diff
@@ -65,6 +65,9 @@ module ExtraTopLevelOperators =
     [<CompiledName("PrintFormatToString")>]
     val sprintf: format: Printf.StringFormat<'T> -> 'T
 
+    [<CompiledName("FailWith")>]
+    val failwith: s: string -> 'T
```

Comment:
This feels strange.
Why did it not create an issue when it was missing? Anyone having any idea?

Reply:
We have it **also** in prim-types (and it is inline there) , so this one should be possible to remove.